ml_backends/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+services:
+  example:
+    build:
+      context: ./example
+    volumes:
+      - ./example/:/app:z
+    ports:
+      - "2000:2000"
+  ngrok:
+    image: ngrok/ngrok:latest
+    ports:
+      - "4040:4040"
+    command:
+      - "http"
+      - "http://host.docker.internal:2000"
+      - "--request-header-add='ngrok-skip-browser-warning': '69420'"
```

Comment:
This request header should be added to the actual API call you are making with `axios`
```json
axios.get('https://your-ngrok-url.ngrok.io/api/endpoint', {
      headers: {
        'ngrok-skip-browser-warning': 'true'
      }
  ```